install/ansible/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
```

Comment:
We should change these to `#!/bin/bash` if we're using `>&`.  It's not available in sh